src/components/organisms/SideNav/index.tsx

```diff
@@ -0,0 +1,64 @@
+import styled from "@emotion/styled";
+import React from "react";
+import theme from "../../../themes/index";
+import IconTypography, {
+  IconTypographyProps,
+} from "../../molecules/IconTypography";
+
+interface SideNavProps {
+  contentList: IconTypographyProps[];
+}
+
+const Container = styled("div")({
+  display: "flex",
+  flexDirection: "column",
+  width: '6.25rem',
+  height: '44rem',
+  backgroundColor: theme.palette.structural[50],
+  gap: `1.25rem`,
+  alignContent: "center",
+  boxShadow: `0px 0px 4px 0px ${theme.palette.grey[200]}`,
+  paddingTop: `1.5625rem`,
+  paddingLeft: 0,
+  paddingRight: 0,
+});
+
+const TemplateStyled = styled("div")<{ colored?: boolean }>(({ colored }) => ({
+  backgroundColor: colored
+    ? theme.palette.success.light
+    : theme.palette.structural[50],
+  color: colored ? theme.palette.primary.main : theme.palette.text.secondary,
```

Comment:
instead of checking the condition for colored twice, check for once and return based on its truthness. like colored {<obj>}: {<obj>}